api/src/Entity/ContentNode.php

```diff
@@ -36,7 +36,8 @@
         'get' => ['security' => 'is_granted("CAMP_COLLABORATOR", object) or is_granted("CAMP_IS_PROTOTYPE", object)'],
     ],
     denormalizationContext: ['groups' => ['write']],
-    normalizationContext: ['groups' => ['read']]
+    normalizationContext: ['groups' => ['read']],
+    order: ['root.id', 'parent.id', 'slot', 'position']
```

Comment:
ok, they changed something about the order of the Entity Objects?

Reply:
I started to get some weird inconsitencies on the item order, sometimes on the server, sometimes locally. Couldn't ping it down to something specific, so thought defining a clear order is the best way to solve it.

Stumbled over this already in https://github.com/ecamp/ecamp3/pull/2649/files#diff-93cc61a23d08cab7232453b3406546f4512315d55c95ecb04aa458e10a7395ac and copied from there